bio/megahit/wrapper.py

```
@@ -0,0 +1,95 @@
+        output_file = snakemake.output.contigs
+    else:
+        output_file = snakemake.output[0]
+
+    output_dir = os.path.split(output_file)[0]
+
+
+# parse params
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+if hasattr(snakemake.resources, "mem_mb"):
```

Ruff (0.11.9)

Undefined name `snakemake`

(F821)